recipes/chktex/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "chktex" %}
+{% set version = "1.7.6" %}
+{% set am_version = "1.15" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://download.savannah.gnu.org/releases/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8ac0e5ca213b2012d44c28f9e4feb9783df44750eb0c30a237d81ff58ef34c8d
+  patches:
+    - 000-makefile-no-dvi.patch
+  # TODO: figure out how to get osx to use ncurses
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - /usr/lib/libncurses.5.4.dylib  # [osx]
```

Comment:
@conda-forge/help-c-cpp I'm having some issues getting `libncurses*.dylib` to be picked up on OSX... any thoughts? Seems like it should just be another flag to `configure`, but it's eluding me...

Reply:
Does it have to do with the version number in the file name?

Reply:
yeah, i guess i need to just dig harder into all the `configure` tooling... all the `r-*` recipes call out _this version in this location_ in specific, and it's somehow available in `sysroot`, but I don't know where that comes from...

Reply:
Maybe this would go away if you also added `--disable-posixre` to the configure line on Mac?  This is the only thing I could think of that would be pulling in system libraries.  Alternatively.  You could add a `export CFLAGS="${CFLAGS} -l$PREFIX/lib/libnurses${SH_EXT}"` or something to explicitly link against the right ncurses